test/unit/spec/common/simpleLogViewer-spec.js

```diff
@@ -951,9 +951,6 @@ define([
                         input = generateInput(jobId);
                     let acc = -1;
 
-                    // // set the job manager's status request loop going
-                    this.jobManager.restoreFromSaved();
-
```

Comment:
move this later in the function so now we capture the first message (which requests a status update for the whole batch)